Lib/test/test_ast.py

```diff
@@ -43,12 +44,36 @@ def to_tuple(t):
     "def f(**kwargs): pass",
     # FunctionDef with all kind of args and docstring
     "def f(a, b=1, c=None, d=[], e={}, *args, f=42, **kwargs): 'doc for f()'",
+    # Decorated FunctionDef
+    textwrap.dedent("""
+    @deco
```

Comment:
Since you use `textwrap.dedent()` it would clearer if add an indentation of the inner code.

Reply:
Please increase the indentation of these fragments.